packages/dev/addons/src/lottie/lottiePlayer.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * Configuration options for the Lottie animation player.
+ */
+export type AnimationConfiguration = {

```

Comment:
Maybe it is just preference, but I would declare this with everything being required, and then the options that are passed in are just `Partial<AnimationConfiguration>`. Then everywhere else where you use `Required<AnimationConfiguration>` can just be `AnimationConfiguration`.